### ISSUE TITLE
Upload data via the API

### ISSUE THREAD

**Issue by peterdesmet** [ices-tools-dev/esas#122]
@JochemBoersma, relating to your comment in https://github.com/ices-tools-dev/esas/issues/110#issuecomment-1262805059, I'm not sure @nicolasvanermen has extensively tested **uploading** data via the API. @nicolasvanermen have you? Or were you using the user interface in the browser?

Note: I have tested downloading data via the API, which works fine.

**Comment by nicolasvanermen** [comment]
No, I have indeed been using the interface only, not being familiar with the API part of the application.

**Comment by peterdesmet** [comment]
That depends on how quickly @JochemBoersma can get the necessary credentials and test this upload feature. In my opinion, we should not wait for that, as it might delay things further and (I assume) test data uploaded by @JochemBoersma can be deleted from the database at any point?

**Comment by cmspinto** [comment]
Yes, it can always be deleted. 
I also assume that he will upload files for a distinct institution of the ones in the database?

**Comment by JochemBoersma** [comment]
Nice to hear that everything what I will upload, can be deleted. It makes it easier to play around :).
Indeed: do not wait for 'my' uploads. But nevertheless, it is a very useful use case. And I really like to assist in testing this feature.

I requested a login at advice@ices.dk (how long does that takes, typically?), and will prepare the datasets which we have obtained.

**Comment by cmspinto** [comment]
@JochemBoersma, you have a user now, please let us know if you need any further help.

Just one highlight on thing: you can submit the data for any institution but you can only resubmit data for your institution. Would you like us to give you permissions to resubmit to any institution?

**Comment by nicolasvanermen** [comment]
Just to notify that one of course should have some kind of approval of the institutes before uploading their data... In case anyone would ever ask ;-)
